pkg/clients/builder.go

```diff
@@ -84,6 +84,7 @@ func (b *Builder) BuildOrDie(ctx context.Context) *Clients {
 		var err error
 		guestKubeConfig, err = client.GetKubeConfigOrInClusterConfig(b.guestKubeConfigFile, nil)
 		if err != nil {
+			klog.Errorf("error reading guesKubeConfig from %s: %v", b.guestKubeConfigFile, err)
 			panic(err)
```

Comment:
Would be `klog.Fatalf` + no panic better? I don't know how the actual output looks like here. This applies to all panics below.

Reply:
fixed